client/js/components/AudioPlayer/AudioPlayer.js

```diff
@@ -0,0 +1,104 @@
+import React, {Component} from 'react';
+import {StyleSheet, View, Button} from 'react-native';
+import Player from 'react-native-streaming-audio-player';
+import SoundPlayer from 'react-native-sound-player';
+import {NativeModules, NativeEventEmitter, Platform} from 'react-native';
+const {RNSoundPlayer} = NativeModules;
+
+export default class AudioPlayer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {currentTime: 0};
```

Comment:
whats the currentTime in the state doing?

Reply:
Removed from this branch